ingestion-lambda/src/processEmailContent.test.ts

```diff
@@ -0,0 +1,62 @@
+import { parseEmail } from './processEmailContent';
+
+describe('parseEmailBody', () => {
+	it('should parse a valid email body', async () => {
+		const emailData = sampleMimeEmailData();
+		const { from, subject, text, date } = await parseEmail(emailData);
+		expect(from).toBe('"Correspondent A" <correspondent.a@example.com>');
+		expect(subject).toBe('Test subject line');
+		expect(text).toContain('This is some test copy: hello world');
+		expect(date).toBe('Wed, 16 Jul 2025 08:38:29 GMT');
+	});
+});
+
+export function sampleMimeEmailData() {
```

Comment:
Since this is returning static data I think this can be a const here?

Reply:
good call -- my reason for making it a function was just so that it could go at the bottom of the file while I was developing, but , but I agree that a `const` is more expressive of the intent so I've updated it now, exporting from its own file as it's actually consumed in two places now 👍👍